server/lib/Room.js

```diff
@@ -493,16 +493,13 @@ class Room extends EventEmitter
 		const previousMaxBitrate = this._maxBitrate;
 		let newMaxBitrate;
 
-		if (numPeers <= 2)
+		if (numPeers < 2)
```

Comment:
I prefer this to be `<= 2` so when there are 2 participants, they both get the maximum bitrate.